src/test/ui/expr/if/if-let.stderr

```diff
@@ -1,8 +1,8 @@
 warning: irrefutable `if let` pattern
-  --> $DIR/if-let.rs:6:13
+  --> $DIR/if-let.rs:6:16
    |
 LL |               if let $p = $e $b
-   |               ^^^^^^^^^^^^^^^^^
+   |                  ^^^
```

Comment:
Span comes from AST and for some reason, macro tokens definitions aren't being collected . Normal `let pat = expr` span don't crop.